src/drivers/PortFolder/test/ExtendedDeprecatedApis.sarif

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
This pattern seems to repeat a lot, is it possible to generate the files from a template instead of having a bunch of the same file?

Reply:
These files are the de-facto "baseline" for each individual test.  At the moment they're all identical, but if we were to make a checkin that caused a regression (but we didn't have time to fix the regression, or had a good reason not to) they would be updated with basic regression info.  So they do have meaningful individual information, just... not right now.  Once we have a better baselining system set up we can eliminate them.